### ISSUE TITLE
Add more info about swag (like shirt sizes) to the checkin dialog

### ISSUE THREAD

**Issue by RobRuana** [magfest/ubersystem#2562]
When we're checking in someone who needs a shirt, the checkin dialog lists that the get a shirt, but doesn't list the size. So we have to go into the attendee details to find the shirt size. The checkin dialog should also have a checkbox to mark "received merch". This may only apply to MAGStock, because I think other events do the merch separately.

**Comment by kitsuta** [comment]
Yeah, most other events do merch separately so this should be done in the
Magstock plugin

**Comment by binary1230** [comment]
may apply to magwest too.  not sure though
